### PR TITLE
1) Refactor Card Page Components Navbar Component

### PR DESCRIPTION
**Pull request by varun13169**
1) horizontal card correction
2) Add New Vertical Card
3) Grid NavBar

### CODE REVIEW

File: pages/authPage/logout.html [hunk 1]
```diff
@@ -12,15 +12,15 @@
 <body>
```
Comment (zoef-yt) [inline pages/authPage/logout.html:12]: UI Changes: For some reason, the navbar is broken do check it out!
image

File: pages/productListingPage/productListingPage-vert.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
+@import "/styles.css";
+
+:root {}
+
+body {}
+
```
Comment (zoef-yt) [inline pages/productListingPage/productListingPage-vert.css:6]: can be removed if not being used.

**Review by zoef-yt — CHANGES_REQUESTED** [review]
The code looks but the UI needs to be fixed.

File: pages/authPage/logout.html (hunk 1 above)
Comment (varun13169) [reply pages/authPage/logout.html:12]: Navbar component is broken due to changes in styles CDN.
this is fixed in the next PR with react.

File: pages/productListingPage/productListingPage-vert.css (hunk 2 above)
Comment (varun13169) [reply pages/productListingPage/productListingPage-vert.css:6]: This is a placeholder that will be used in react app

**Review by MayankKumar10 — APPROVED** [review]
Look into your UI everything else Looks good to me you can merge it.

**Review by Rohanmond — APPROVED** [review]
except the ui everything looks good

**Pull request merged**